lib/helpers.js

```diff
@@ -1,14 +1,18 @@
 'use strict';
 
+// BUG: quick fix
+Error.captureStackTrace = () => { };
```

Comment:
What bug is this fixing? We can't modify the builtin objects (Error, Number, Math, etc), this could interfere with other libraries in the same environment.